docs/reference/6-data-import/getting-started/installation/readme.md

```diff
@@ -0,0 +1,44 @@
+# Installation
+
+# Before You Install
+
+This section provides general information you should review before installing Ed-Fi Data Import.
+
+## Compatibility & Supported ODS / API Versions
+
+Data Import is designed for use with the Ed-Fi ODS / API v5.3+. See the [https://edfi.atlassian.net/wiki/spaces/TT/pages/18645319](https://edfi.atlassian.net/wiki/spaces/TT/pages/18645319) and [Ed-Fi Technology Version Index](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/20875717/Ed-Fi+Technology+Version+Index) sections for more details on the ODS / API and version compatibility.
```

Comment:
First link should be to `[Ed-Fi ODS/API Platform](../../../ods-api-platform/readme.mdx)`, second to `[Ed-Fi Technology Suite Supported Versions](../../../0-roadmap/supported-versions.md)`